contributed_definitions/NXellipsometry.nxdl.xml

```diff
@@ -183,35 +141,18 @@ A draft version of a NeXus application definition for ellipsometry.-->
                 <item value="ultraviolet spectroscopic ellipsometry"/>
                 <item value="uv-vis spectroscopic ellipsometry"/>
                 <item value="NIR-Vis-UV spectroscopic ellipsometry"/>
-                <item value="imaging ellipsometry"/>
```

Comment:
Why is imaging ellipsometry removed? It is different in the sense to use a single angle, but an important application for detecting microstructure in films which are too often are assumed to be homogeneous.

Reply:
with the new NXopt, it is still possible to define "imaging ellipsometry", by using the "experiment_sub_type" as "imaging" the "experiment_type" as "ellipsometry".